qadence/backends/pulser/config.py

```diff
@@ -49,6 +50,12 @@ class Configuration(BackendConfiguration):
     # interaction type
     interaction: Interaction = Interaction.NN
 
+    # platform to execute sequence on on the cloud
+    cloud_platform: Optional[EmulatorType] = None
```

Comment:
Not sure that `cloud_platform` is the best naming choice here? Maybe `cloud_backend` or `cloud_device`?